chainsaw-proto/src/grpc/mod.rs

```diff
@@ -1,10 +1,6 @@
 pub mod google {
-    pub mod protobuf {
-        include!("google.protobuf.rs");
-    }
+    pub mod protobuf;
 }
 pub mod helloworld {
-    pub mod v1 {
-        include!("helloworld.v1.rs");
-    }
+    pub mod v1;
```

Comment:
Is this broken now or just slightly magic?

Reply:
The `include!()` macro just replaces the macro call with the specified file contents I believe, so `grpc-build` just doesn't bother with that and actually uses the file as a module. It builds and runs as far as I can see.